### ISSUE TITLE
Can't play from locally downloaded file

### ISSUE THREAD

**Issue by ishabo** [johnsonsu/react-native-sound-player#10]
Hi,

I have a file downloaded this mp3: https://ia802508.us.archive.org/5/items/testmp3testfile/mpthreetest.mp3 using `react-native-fs` into `MAIN_BUNDLE` on an IOS emulator. I can verify the file is there and I can play it with any audio player on my Mac `/Library/Developer/CoreSimulator/Devices/594B5EDF-5E73-4040-9236-2DA23A1BAC86/data/Containers/Data/Application/A60A1E1B-EBD9-4A59-9B90-4A3148D68D7D/Documents/mpthreetest.mp3`

Yet, I get this error when I run: 
```javascript
SoundPlayer.playSoundFile(filename, 'mp3');
```

![image](https://user-images.githubusercontent.com/1992039/44851938-cad7b080-ac59-11e8-8b0b-baf548f652cc.png)

Looking at `RNSoundPlayer.m` I can see that the default location is `mainBundle`. I tried with many files mp4, ogg but the same is happening.

When I play the URL directly with 
```javascript
SoundPlayer.playUrl(url)
```
it plays without a problem, but I need to download files first before playing it to make sure the file is available offline.

Any help would be appreciated!

**Comment by nu5rim3** [comment]
how to play downloaded sound files. am looking for a solution for the same problem.

**Comment by istvan-szilagyi** [comment]
Well it seems you're a bit confused with the directories :)
1. Main Bundle: this directory is read-only, so you CAN'T download files here within the application, you can see only files that you have builded into the bundle.
2. Documents directory: this directory has read-write permissions. As I can see in your path, you have downloaded the file into this directory. The command `SoundPlayer.playSoundFile` can play only files from the bundle, and not Documents directory. This is the code in the Xcode project:
`NSString *soundFilePath = [[NSBundle mainBundle] pathForResource:name ofType:type];`

Actually I could make it run from documents, but you will have to edit the Xcode project of this module:
1. Open the Xcode project
2. Open RNSoundPlayer.m
3. Find function: RCT_EXPORT_METHOD(playUrl:(NSString *)url)
4. After line: `NSURL *soundURL = [NSURL URLWithString:url];` add the following:
```
if (soundURL == nil) {
        soundURL = [NSURL fileURLWithPath:url];
    }
    if (soundURL == nil) {
        return;
    }
```

Now just call (from the React project):
SoundPlayer.playUrl('/Library/Developer/CoreSimulator/...<your path to the file>');

Cheers!

**Comment by johnsonsu** [comment]
This should be resolved by #37, thanks to @assertchris.
